src/comon/Sidebar/Sidebar.styled.js

```diff
@@ -0,0 +1,91 @@
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import styled from '@emotion/styled';
+
+export const StyledBox = styled(Box, {
+  shouldForwardProp: (p) => p !== 'showText',
+})`
+  display: flex;
+  flex-direction: column;
+  gap: ${(p) => (p.showText ? '8px' : '24px')};
+  box-sizing: border-box;
+`;
+
+export const ActiveLink = styled(Link, {
+  shouldForwardProp: (p) => p !== 'showText',
+})`
+  padding: 16px;
+  width: 290px;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  gap: 8px;
+  box-sizing: border-box;
+  font-size: ${(p) => p.theme.typography.h5.fontSize};
```

Comment:
Use just `${(p) => p.theme.typography.h5}` without `font-size:`, it will pull up the entire object for the font at once